src/components/layout/nav.js

```diff
@@ -110,7 +110,7 @@ export const Wrapper = styled(Container)`
     justify-content: space-between;
     height: 7.2rem;
     @media ${device.laptopL} {
-        width: 90%;
+        width: ${({ width }) => width ?? "90%"};
```

Comment:
Did you meant to use the `||` operator?

Reply:
`??` This notation is to add limitation to our code that we must not have an empty value for props.